src/Microsoft.Azure.WebJobs.Extensions.Storage/StorageLoadBalancerQueue.cs

```diff
@@ -111,6 +111,7 @@ private CloudQueue Convert(string queueMoniker)
                 sharedWatcher: _sharedWatcher,
                 queueOptions: _queueOptions,
                 queueProcessorFactory: _queueProcessorFactory,
+                functionDescriptor: new FunctionDescriptor(),
```

Comment:
I'm pretty sure that this isn't used anywhere anymore -- but even if so, I believe it was meant to be a "backplane" for queues across our system, so having (null) in the function name would make sense. I didn't want to branch the logging logic based on this, though -- so leaving as-is for now. If we end up seeing this as a problem, we can address it with a follow-up.